python/vowpalwabbit/sklearn_vw.py

```diff
@@ -866,6 +866,10 @@ def _more_tags(self):
 class VWRegressor(VW, RegressorMixin):
     """Vowpal Wabbit Regressor model """
 
+    def __init__(self, **kwargs):
+        kwargs["convert_labels"] = False
```

Comment:
Should this only be set if it wasn't explicitly provided by the caller?

Reply:
Yes you're right.